src/fluid/fluid.cpp

```diff
@@ -567,10 +567,10 @@ TaskStatus CalculateFluidSourceTerms(MeshBlockData<Real> *rc,
           }
           src(ceng, k, j, i) = gdet * alpha * (Ta - TGam);
 #else
-          SPACETIMELOOP2(mu, nu) {
-            TGam += Tmunu[mu][nu] * gam[nu][0][mu];
-          }
-          src(ceng,k,j,i) = gdet * TGam;
+                         SPACETIMELOOP2(mu, nu) {
+                           TGam += Tmunu[mu][nu] * gam[nu][0][mu];
+                         }
+                         src(ceng, k, j, i) = gdet * TGam;
```

Comment:
I don't know what's going on with `clang-format-12` here

Reply:
Ugh I really hate doing that... maybe as code changes this formatting bug will go away later